### PR TITLE
Add blog post on the French Open Source Software Community Award

### PR DESCRIPTION
**Pull request by francoisgoupil**
Here is a suggested blog post in the "Events" category about the recent prize the community received by the French Government. 
Feel free to give feedback and correct some of my English mistakes.

### CODE REVIEW

File: _posts/2022-02-05-frenchaward.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "An Open Source Software Award for scikit-learn"
+date: January 8, 2022
```
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:3]:
```suggestion
date: February 7, 2022
```

File: _posts/2022-02-05-frenchaward.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+categories:
+  - Events
+tags:
+  - Open Source
+  - Award
+  - Community
+  - French government
```
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:10]:
```suggestion
  - Open Source
  - Press
```

File: _posts/2022-02-05-frenchaward.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: "An Open Source Software Award for scikit-learn"
+date: January 8, 2022
+categories:
+  - Events
```
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:5]:
```suggestion
  - Press
```

File: _posts/2022-02-05-frenchaward.md (hunk 2 above)
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:10]:
```suggestion
  - Open Source
  - Community
```

Comment (reshamas) [reply _posts/2022-02-05-frenchaward.md:10]: @francoisgoupil 
I think it's good to limit to a couple of tags that we already have.

File: _posts/2022-02-05-frenchaward.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    website: https://www.linkedin.com/in/fran%C3%A7ois-goupil/
+    image: francois_goupil.jpeg 
+---
+
+<div>
+  <img src="/blog/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+We are pleased to announce that scikit-learn has received a prize for open-source scientific software from the French government. It is great recognition for all the community of contributors and users of a project born in France. Congratulation to the worldwide community for this great achievement!
```
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:24]:
```suggestion
We are pleased to announce that scikit-learn has received a prize for open-source scientific software from the French government. It is great recognition for all the community of contributors and users of a project born in France. Congratulations to the worldwide community for this great achievement!
```

File: _posts/2022-02-05-frenchaward.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+>“I literally owe my career in the data space to scikit-learn. It’s not just a framework but a school of thought regarding predictive modeling. Super well deserved, folks :) “ 
+Maykon Schots from Brasil
+
+>“Well done everyone for getting us here :)” 
+Joel Nothman from Australia
+
+__About the Award__
+
+For the first year, the Ministry of Higher Education, Research and Innovation awarded the Open Science Prizes for Free Research Softwares. Ten software developed by French teams were rewarded for their contribution to the advancement of scientific knowledge.
```
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:39]: @francoisgoupil 
Can you add a few links to these items?  Thank you.

Comment (francoisgoupil) [reply _posts/2022-02-05-frenchaward.md:37]: The link below the article is a reference this section. I think it is enough

File: _posts/2022-02-05-frenchaward.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+As part of the second National Plan for Open Science, the Open Science Awards for Free Research Software highlights projects and research teams working on the development and dissemination of free software. It aims to emphasize teams and projects contributing to the construction of a common good.
+
+The main goal of this award is to draw the attention of the scientific community to exceptional or very promising achievements, which can serve as a model for the next generations of researchers and engineers. The prizes were awarded on the decision of a jury of experts rendered by Daniel Le Berre (Lens Computer Science Research Center, University of Artois-CNRS).
+
+The awards came in three categories, which recognized: 
+- The scientific and technical quality of the software
+- Building an active community of contributors and users
+- The essential effort to provide documentation that facilitates the use and appropriation of the software
+See the whole list of award recipients:
```
Comment (reshamas) [inline _posts/2022-02-05-frenchaward.md:46]:
```suggestion
See the whole list of award recipients. This article is available in both French and English.
```

**Pull request merged**